spvenvelope.go

```diff
@@ -0,0 +1,153 @@
+package bc
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/libsv/go-bk/envelope"
+	"github.com/libsv/go-bt"
+)
+
+// SPVEnvelope is a struct which contains all information needed
```

Comment:
Might be a good idea to add the TSC spec to this comment as well https://tsc.bitcoinassociation.net/standards/spv-envelope/